Sources/SwiftGRPCNIO/GRPCError.swift

```diff
@@ -87,8 +87,11 @@ public enum GRPCServerError: Error, Equatable {
   /// It was not possible to serialize the response protobuf.
   case responseProtoSerializationFailure
 
+  /// Zero requests were sent for a unary-request call.
```

Comment:
Will this always be zero? If so, maybe the error should be `noRequestsSent`

Reply:
Renamed to `noRequestsButOneExpected` (to make it clear what the expectation is); let me know if that works for you.

Reply:
sgtm